core/Cargo.toml

```diff
@@ -97,6 +97,7 @@ twox-hash = "1.6.3"
 
 [features]
 default = []
+nightly = []
```

Comment:
We have some references to this feature in our code and get warnings when we run miri (which requires use of nightly Rust)